### PR TITLE
[Map/Scatter/Timeline] Add chart footer to hide denom field

### PR DESCRIPTION
**Pull request by chejennifer**
- create a new shared chart footer component that expands on click to show the options for the chart. 
- remove default "Count_Person" denominator from all tools
- update labels to not say "Per Capita" when ratio is selected (because now, user can input any stat var for the denominator and per capita would not be accurate)
- move stat var chips in timeline to the top of the chart

Clean up:
- remove unused styling from stylesheets for map/scatter/timeline
- clean up old params from info page links

screen-recording-_16_
screen-recording-_15_
screen-recording-_14_

### CODE REVIEW

**Review by beets — APPROVED** [review]
nice change! thanks for the cleanup and bugfixes too

File: static/css/chart_footer.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Styling for chart footer component.
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply static/css/chart_footer.scss]: done

File: static/js/shared/chart_footer.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+ */
+
+import _ from "lodash";
+import React, { useState } from "react";
+import { FormGroup, Input, Label } from "reactstrap";
+
+import { SourceSelector, SourceSelectorSvInfo } from "./source_selector";
+import { urlToDomain } from "./util";
+
+interface ChartFooterPropType {
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply static/js/shared/chart_footer.tsx]: done

File: static/js/tools/scatter/plot_options.tsx [hunk 3]
```diff
@@ -39,211 +32,232 @@ import {
...
+                  onKeyDown={(e) => {
+                    if (e.key === "Enter") {
+                      y.setDenom(yDenomInput);
+                    }
+                  }}
+                  type="text"
+                  value={yDenomInput}
+                  onChange={(e) => setYDenomInput(e.target.value)}
+                  disabled={!y.value.perCapita}
+                  placeholder="Enter a stat var dcid eg. Count_Person"
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply static/js/tools/scatter/plot_options.tsx]: done

**Pull request merged**